docs/index.rst

```diff
@@ -0,0 +1,29 @@
+Welcome to plankton's documentation!
+====================================
+
+Plankton is a Python package that makes it easy to develop complex stateful device simulations. It
+is licensed under GPL version 3 and the source is available on github_, where you are welcome to
+open new issues so the package can improve.
+
+If you're new to Plankton reading the :ref:`user_guide` is a good start. Once you plan on writing
+your own device it's time to go through the :ref:`developer_guide`. The :ref:`api_documentation`
+can also help with that.
+
+Documentation contents:
+
+.. toctree::
+   :maxdepth: 2
+
+   user_guide/index
+   developer_guide/index
+   api/index
+
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
```

Comment:
Not sure if we can/need to do anything about this, but this leads to an alphabetized module index where every module is listed under one entry: `p - plankton`. Might be slightly better, if it's possible, to ignore the `plankton` and list them as `a - adapters`, `c - core`, etc?